src/main/java/de/fearnixx/jeak/teamspeak/voice/sound/opus/OpusParameters.java

```diff
@@ -0,0 +1,54 @@
+package de.fearnixx.jeak.teamspeak.voice.sound.opus;
+
+/**
+ * This class was extracted from an example by manevolent
```

Comment:
Pulled completely or just aligned? If pulled completely, we should add a proper reference to the original file.